### ISSUE TITLE
Table with subtable and colSpan renders text in incorrect positions - EDIT: also with columns

### ISSUE THREAD

**Issue by Gazzell** [bpampuch/pdfmake#668]
If I insert a table in the first cell of another table, with colSpan and rowSpan, the second rows texts are rendered in wrong positions although their style is well rendered. In this example, the texts Ex4 ans Ex5 should be rendered in the red cells under Ex2 and Ex3:

```js
var dd = {
content: [
    {
        table: {
            widths:  [ '*',30, 40,'*', 30, 40 ],
            body: [
                [
                    {
                        table: {
                            widths: [ '*' ],
                            body: [
                                [
                                    ''
                                ]
                            ]
                        },
                        colSpan: 4,
                        rowSpan: 2
                    },
                    '',
                    '',
                    '',
                    {
                        text: 'Ex2',
                        fillColor: 'red'
                    },
                    {
                        text: 'Ex3',
                        fillColor: 'red'
                    }
                ],
                [
                    '',
                    '',
                    '',
                    '',
                    {
                        text: 'Ex4',
                        fillColor: 'red'
                    },
                    {
                        text: 'Ex5',
                        fillColor: 'red'
                    }
                ]
            ]
        }
    }
] } 
```

EDIT. It looks it also happens with columns. It has something to do with using rowSpan in table cells that have tables or columns:

```js
var dd = {
content: [
    {
        table: {
            widths: [ '*',30, 40,'*', 30, 40 ],
            body: [
                [
                    {
                        columns:[
                            'PP1'
                        ],
                        colSpan: 4,
                        rowSpan: 2
                    },
                    '',
                    '',
                    '',
                    {
                        text: 'Ex2',
                        fillColor: 'red'
                    },
                    {
                        text: 'Ex3',
                        fillColor: 'red'
                    }
                ],
                [
                    '',
                    '',
                    '',
                    '',
                    {
                        text: 'Ex4',
                        fillColor: 'red'
                    },
                    {
                        text: 'Ex5',
                        fillColor: 'red'
                    }
                ]
            ]
        }
    }
]
}
```

**Comment by devmiegsi** [comment]
First of all, congratulations on the project @liborm85 .
Second, do you have any news or expected date to be solved? i need to find a solution or workaround for this bug to finish a project.

**Comment by liborm85** [comment]
Fixed by PR https://github.com/bpampuch/pdfmake/pull/2764.
Released in version 0.2.11.
